src/dotty/tools/dotc/repl/CompilingInterpreter.scala

```diff
@@ -738,6 +797,10 @@ class CompilingInterpreter(out: PrintWriter, ictx: Context) extends Compiler wit
     INTERPRETER_LINE_PREFIX + num
   }
 
+  /** get the current line name */
```

Comment:
Capital letter at the start of a doc comment, but here the method is private and the comment has no information that's not in the method name, so I just wouldn't put a comment